Header header

bool dvl_sts
bool svs_sts
bool fog_sts
bool nav_sts
bool bat_sts
bool t_sts
bool h_sts
bool p_sts
bool water_sts

